portability-types-transfer/src/main/java/org/dataportabilityproject/types/transfer/models/contacts/ContactsModelWrapper.java

```diff
@@ -0,0 +1,21 @@
+package org.dataportabilityproject.types.transfer.models.contacts;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+
+/**
+ * A collection of contacts as serialized vCards.
+ */
+public class ContactsModelWrapper extends DataModel {
```

Comment:
I think in willard's PR, we changed the *Wrapper classes to *ContainerResource

Reply:
OK, I was going to make all of the suggested changes and then rebase my commits against the upstream.

Reply:
OK I'll rebase my commits once I merge the upstream in